### DISCUSSION TITLE
Install of Security Onion hangs on an HP Elite mini 800 G9

### DISCUSSION THREAD

**Discussion by megatronGA** [Security-Onion-Solutions/securityonion#12103] [General]
I'm having issues installing Security Onion on an HP Elite 800 G9. Installing via a USB thumb drive ISO and and it hangs. Is there a workaround for this?

**Comment by dougburks** [comment]
Did you verify the ISO image as shown at https://securityonion.net/download?

How did you create the USB drive? Did you use Balena Etcher as shown at https://docs.securityonion.net/en/2.4/download.html?

Have you tried the steps at https://docs.securityonion.net/en/2.4/trouble-booting.html?
